src/System.Net.Http/tests/StressTests/HttpStress/Program.cs

```diff
@@ -500,6 +521,17 @@ void ValidateContent(string expectedContent, string actualContent)
                         // Echos back the requested content in a full duplex manner.
                         await context.Request.Body.CopyToAsync(context.Response.Body);
                     });
+                    endpoints.MapPost("/speed", async context =>
```

Comment:
We shouldn't need a separate /speed endpoint for this.  This can just be used instead of the existing /duplex endpoint, which is functionally the same; this just explicitly controls the buffer size.